docs/source/ref/urls.rst

```diff
@@ -22,7 +22,7 @@ has one, and its draft version will be used otherwise.
   `parse_dandi_url()` converts this format to a `DandisetURL`.
 
 - Any ``https://gui.dandiarchive.org/`` or
-  ``https://*dandiarchive-org.netflify.app/`` URL which redirects to
+  ``https://*dandiarchive-org.netlify.app/`` URL which redirects to
```

Comment:
Does this need to correspond with something lincbrain specific -- e.g. `https://staging--lincbrain-org.netlify.app/`

Reply:
Thats a good point.  I was just trying to make `codespell` happy.  There seems to be lots of references to DANDI in this docs file.  Since we will be working towards abstracting the `dandi-cli`/`linc-cli` this year, I would suggest that we hold off to converting this file to be lincbrain specific.  What do you think?